papis/bibtex.py

```diff
@@ -342,8 +342,6 @@ def to_bibtex(document: papis.document.Document, *, indent: int = 2) -> str:
                     "Key '%s' is not present for ref '%s'",
                     journal_key, document["ref"])
 
-            bib_value = string_to_latex(bib_value)
```

Comment:
I added this because I had some journal named `"Something & Something"` and the resulting bib file would be invalid. Is there a better way to fix that?

Reply:
I would say ah, I see, good question. I can think of two ways of going around it, since this might happen not only with journal names but also with titles and so on.

I have been thinking about setting an overridable field for an exporter. I.e,
for instance you might have

```yaml
title: Dr. Jêkyll & Mr. Hÿdë
title_bibtex: Dr. Jêkyll \& Mr. Hÿdë
```

and then if you have in your library unicode activated because you use
xelatex or whatever, then the bibtex exporter should prefer
`title_bibtex` over `title` checking if there is a
`(key_name + "_bibtex") in document`.

This is a way of overriding problematic cases, not ideal and potentially
a lot of hand work but at least it's quite special-cases-proof.

The other option is to identify which characters are problematic in the *tex
family when copied /as-is/ to the bibtex. I can think of &$\( ...
So it would be quite a mess to list all options I think.

Reply:
Hm, yeah, I found another project trying to escape some basic latex strings
https://github.com/JelteF/PyLaTeX/blob/1a73261b771ae15afbb3ca5f06d4ba61328f1c62/pylatex/utils.py#L14-L30
but not sure that's complete in any way. Your idea seems more general and pretty straightforward, so it sounds good to me!

Reply:
that's interesting, I'll go ahead an implement (and document) the overriding functinality, which is as you say a minimal change and can help some folks get things done quick and dirty ;)